src/App.tsx

```diff
@@ -1,55 +1,163 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
+// eslint-disable-next-line import/no-duplicates
+import IconChevronDown from '@mdi/react';
+import { mdiChevronDown, mdiClose } from '@mdi/js';
+// eslint-disable-next-line import/no-duplicates
+import IconClose from '@mdi/react';
 import './App.scss';
+import classNames from 'classnames';
 import { peopleFromServer } from './data/people';
+import { DropdownList } from './components/DropdownList';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [selectedPerson, setSelectedPerson]
+    = useState<Person | null>(peopleFromServer[0]);
```

Comment:
you must not select person by default 
![image](https://github.com/mate-academy/react_autocomplete/assets/96265844/efc0febc-b44a-4c0d-8ccd-7cae40889add)